plotcostsAnalytic.py

```diff
@@ -0,0 +1,118 @@
+# plotcostsAnalytic.py
```

Comment:
I noticed that this file just got added to the folder in the root repository. Is it supposed to be here?

Reply:
Sorry.  It should replace the function with the same name in xscape.  I've updated that from the original pull request to fix a bug in the way that the legend was displayed.

Reply:
I think you have already modified the file at `xscape/plotcostsAnalytic.py`.

Reply:
Is that the same file as this one?